### ISSUE TITLE
Image changes not reflecting in playground on dev

### ISSUE THREAD

**Issue by Bobakanoosh** [harlan-zw/nuxt-og-image#117]
### Describe the bug

I can get the OG image playground to load, but when I change my template, the changes aren't reflecting. For example if I add some random text to the DOM, save, hard refresh the page, nothing appears. I have to restart the nuxt server every time.

I've tried setting `cache: false` for the time being but that didnt help

### System / Nuxt Info

```Shell
Nuxt project info:                                                                                                         5:29:47 PM

------------------------------
- Operating System: Windows_NT
- Node Version:     v20.10.0
- Nuxt Version:     3.8.1
- CLI Version:      3.10.0
- Nitro Version:    2.7.2
- Package Manager:  pnpm@8.10.5
- Builder:          -
- User Config:      modules, nitro, device, googleFonts, imports, vue, routeRules, css, devtools, experimental, srcDir
- Runtime Modules:  @pinia/nuxt@0.5.1, nuxt-typed-router@3.3.3, nuxt-og-image@2.2.4, @vueuse/nuxt@10.6.0, @unocss/nuxt@0.57.7, radix-vue/nuxt, nuxt-icon@0.6.6, @nuxtjs/google-fonts@3.1.0, @nuxt/image@1.0.0, @nuxtjs/device@3.1.1
- Build Modules:    -
------------------------------
```

**Comment by harlan-zw** [comment]
Thanks for the issue, this is an upstream bug in Nuxt that I hope will be fixed in the next patch update.

**Comment by serkodev** [comment]
There is a hacky way to temporary resolve the problem. Hope this can help before Nuxt fix.

1. Create a blank `page` and import your `OgImage` from project 
```vue
<!-- pages/ogimage.vue -->
<script setup lang="ts">
import MyOgImage from '~/components/OgImage/MyOgImage.vue'
</script>

<template>
  <div>
    <MyOgImage />
  </div>
</template>
```
2. Start `nuxt dev`
3. Open browser, go to the `ogimage` page (you created on step 1)
4. Create a new tab on browser and go to other page that you want to test with the OgImage
5. It should be able to hot reload

**Comment by harlan-zw** [comment]
This should be solved in Nuxt 3.9.0
